src/main/java/mekanism/common/item/gear/ItemJetpack.java

```diff
@@ -89,12 +99,12 @@ public void setMode(ItemStack stack, JetpackMode mode) {
     @Override
     public void addHUDStrings(List<Component> list, Player player, ItemStack stack, EquipmentSlot slotType) {
         if (slotType == getSlot()) {
-            ItemJetpack jetpack = (ItemJetpack) stack.getItem();
+            final ItemJetpack jetpack = (ItemJetpack) stack.getItem();
             list.add(MekanismLang.JETPACK_MODE.translateColored(EnumColor.DARK_GRAY, jetpack.getMode(stack)));
             GasStack stored = GasStack.EMPTY;
-            Optional<IGasHandler> capability = stack.getCapability(Capabilities.GAS_HANDLER_CAPABILITY).resolve();
+            final Optional<IGasHandler> capability = stack.getCapability(Capabilities.GAS_HANDLER_CAPABILITY).resolve();
             if (capability.isPresent()) {
-                IGasHandler gasHandlerItem = capability.get();
+                final IGasHandler gasHandlerItem = capability.get();
```

Comment:
No need to change these lines to have final given you aren't touching them otherwise.